.github/workflows/code_style.yml

```diff
@@ -0,0 +1,32 @@
+name: Code style
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: 
+          - "3.9"
```

Comment:
The python version in the project 3.10. Here we have version 3.9. Is it critical?